apollo/integrations/db/postgres_proxy_client.py

```diff
@@ -22,9 +23,18 @@ def __init__(self, credentials: Optional[Dict], client_type: str = "postgres", *
                 f"{client_type.capitalize()} agent client requires {_ATTR_CONNECT_ARGS} in credentials"
             )
 
-        self._connection = psycopg2.connect(
+        connect_args = {
             **credentials[_ATTR_CONNECT_ARGS],
-        )
+        }
+        # we were having tcp keep alive issues in Azure, so we're forcing it now unless configured from the dc
+        # https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-KEEPALIVES
+        if _KEEP_ALIVE_SETTING not in connect_args:
+            connect_args[_KEEP_ALIVE_SETTING] = 1
+            connect_args[f"{_KEEP_ALIVE_SETTING}_idle"] = 30
+            connect_args[f"{_KEEP_ALIVE_SETTING}_interval"] = 10
+            connect_args[f"{_KEEP_ALIVE_SETTING}_count"] = 5
+
```

Comment:
Very small nit, but maybe just easier to do keepalives_idle (easier to search later for these settings if needed :)).

Reply:
Yeah, good point, will make the change